res/values/strings.xml

```diff
@@ -3,6 +3,7 @@
 
     <string name="about_android">%1$s Android app</string>
     <string name="about_version">version %1$s</string>
+    <string name="actionbar_search">Search</string>
```

Comment:
i would use here "search in the current folder" to avoid misunderstandings

Reply:
Done